### PR TITLE
Added previously reexported dependencies explicitly.

### PR DESCRIPTION
**Pull request by soerendomroes**
To be merged once https://github.com/eclipse/elk/pull/1074 is released.

### CODE REVIEW

**Comment by HannesWell** [comment]
Thank you for this one. I wanted to propose a similar change with the same resulting changes.

This could be submitted even before https://github.com/eclipse/elk/pull/1074, having more dependencies than (currently) absolutely necessary should not cause any harm.

File: plugins/de.cau.cs.kieler.klighd/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -8,7 +8,8 @@ Bundle-RequiredExecutionEnvironment: JavaSE-11
 Require-Bundle: de.cau.cs.kieler.klighd.kgraph,
  de.cau.cs.kieler.klighd.krendering;visibility:=reexport,
  org.eclipse.core.runtime;bundle-version="3.6.0";visibility:=reexport,
- org.eclipse.elk.core.service;bundle-version="0.1.0",
+ org.eclipse.elk.core,
+ org.eclipse.elk.core.service,
```
Comment (NiklasRentzCAU) [inline plugins/de.cau.cs.kieler.klighd/META-INF/MANIFEST.MF:12]: Does this change in ELK also affect the Maven dependencies in any way? In that case, the corresponding pom.xml (and pom.eclipse.xml and pom.lsp.xml, see their documentation up top) should be adapted as well.

Comment (HannesWell) [reply plugins/de.cau.cs.kieler.klighd/META-INF/MANIFEST.MF:12]: No it doesn't. In Maven all dependencies are transitive automatically.
And one can only prevent that by explicitly excluding certain artifacts. But as far as I can neither KLighD nor ELK excludes dependencies, does it?

**Review by NiklasRentzCAU — APPROVED** [review]
LGTM

**Pull request merged**